maxim/apis/maxim_apis.py

```
@@ -689,45 +696,205 @@ def get_folders(self) -> List[Folder]:
+            
+        Returns:
+            tuple: (file_data, mime_type, size)
+        """
+        try:
+            MAX_FILE_SIZE_BYTES = 100 * 1024 * 1024
+            if isinstance(attachment, FileDataAttachment):
+                file_data = attachment.data
+                mime_type = attachment.mime_type or 'application/octet-stream'
+                size = len(file_data)
+                if size > MAX_FILE_SIZE_BYTES:
+                    raise ValueError(f"File size exceeds the maximum allowed size of {MAX_FILE_SIZE_BYTES} bytes")
```

Ruff (0.12.2)

Abstract `raise` to an inner function

(TRY301)